pkg/util/wait/wait.go

```diff
@@ -82,22 +81,18 @@ func JitterUntil(f func(), period time.Duration, jitterFactor float64, sliding b
 		}
 
 		func() {
-			defer runtime.HandleCrash()
```

Comment:
xref is here: https://github.com/openshift/origin/pull/10526#issuecomment-241059828